pytest_fixtures/satellite_auth.py

```diff
@@ -86,6 +98,27 @@ def auth_source_ipa(module_org, module_loc, ipa_data):
     ).create()
 
 
+@fixture(scope='function')
+def auth_source_open_ldap(module_org, module_loc, open_ldap_data):
+    return entities.AuthSourceLDAP(
+        onthefly_register=True,
+        account=open_ldap_data['ldap_user_name'],
+        account_password=open_ldap_data['ldap_user_passwd'],
+        base_dn=open_ldap_data['base_dn'],
+        attr_firstname=LDAP_ATTR['firstname'],
+        attr_lastname=LDAP_ATTR['surname'],
+        attr_login=LDAP_ATTR['login'],
+        server_type=LDAP_SERVER_TYPE['API']['posix'],
+        attr_mail=LDAP_ATTR['mail'],
+        name=gen_string('alpha'),
+        host=open_ldap_data['ldap_hostname'],
```

Comment:
With the change I mentioned above (again, if needed to be in a fixture), would look like this.
```suggestion
        account=open_ldap_data.username,
        account_password=open_ldap_data.password,
        base_dn=open_ldap_data.base_dn,
        attr_firstname=LDAP_ATTR['firstname'],
        attr_lastname=LDAP_ATTR['surname'],
        attr_login=LDAP_ATTR['login'],
        server_type=LDAP_SERVER_TYPE['API']['posix'],
        attr_mail=LDAP_ATTR['mail'],
        name=gen_string('alpha'),
        host=open_ldap_data.hostname,
```